pkg/core/storage/leveldb_store.go

```diff
@@ -21,8 +22,9 @@ type LevelDBStore struct {
 // NewLevelDBStore returns a new LevelDBStore object that will
 // initialize the database found at the given path.
 func NewLevelDBStore(cfg LevelDBOptions) (*LevelDBStore, error) {
-	var opts *opt.Options // should be exposed via LevelDBOptions if anything needed
+	var opts = new(opt.Options) // should be exposed via LevelDBOptions if anything needed
 
+	opts.Filter = filter.NewBloomFilter(10)
```

Comment:
How about an option in `*.yml` config?

Reply:
Meh. I had this though, bu it's so deep that I'm not sure it's really appropriate. But at the same time it affects memory consumption if nothing else.